engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -242,13 +242,15 @@ private void initRenderGraph() {
         renderGraph.addNode(worldReflectionNode, "worldReflectionNode");
 
         // sky rendering
-        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(new ResourceUrn("engine:sceneReflectiveRefractive"), FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
+        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(RefractiveReflectiveBlocksNode.REFRACTIVE_REFLECTIVE, FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
```

Comment:
The reason I did not import upto REFRACTIVE_REFLECTIVE was because the surrounding code (at around 20 or 30 instances) uses the Node.FBO syntax instead of FBO syntax.